packages/util/src/sha1.ts

```diff
@@ -43,32 +43,32 @@ export class Sha1 {
   /**
    * Holds the previous values of accumulated variables a-e in the compress_
    * function.
-   * @type {!Array<number>}
+   * @type {!number[]}
```

Comment:
Since you're changing these, can you actually remove the `type` comments? They are useless with TS, and can even be misleading.

http://go/tsstyle#comments-documentation

Reply:
Done.